interface/wx/maskededit.h

```diff
@@ -0,0 +1,935 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        maskededit.h
+// Purpose:     interface of wxMaskedEdit
+// Author:      wxWidgets team
+// RCS-ID:      $Id$
+// Licence:     wxWindows license
+/////////////////////////////////////////////////////////////////////////////
+
+/**
+@defgroup group_maskededit_predef Masked edit predefined functions
+*/
+
+/**
+    Struct with the colours used for different cases with wxMaskedEditText and
+    wxMaskedEditCombo.
+
+    If you don't set a member of this struct, the standard system default
+    colour for the control is used for it.
+
+    @library{wxcore}
+    @category{ctrl}
+
+    @see wxMaskedEditText, wxMaskedEditCombo
+
+    @since 3.1.6
+
+*/
+struct wxMaskedEditColours
+{
+    /// Foreground colour used when everything is OK.
+    wxColour colOKForegn;
+    /// Background colour used when everything is OK.
+    wxColour colOKBackgn;
+
+    /// Foreground colour used when something is wrong.
+    wxColour colWrForegn;
+    /// Background colour used when something is wrong.
+    wxColour colWrBackgn;
```

Comment:
Here, additionally, "Wr" should be replaced with something more clear. My candidates are "Bad" or "Error" (maybe "Err"). Maybe "Invalid" (or "Inv"). But definitely not "Wr".